src/snmalloc/backend/backend.h

```diff
@@ -23,9 +23,6 @@ namespace snmalloc
     using Pal = PAL;
     using SlabMetadata = typename PagemapEntry::SlabMetadata;
 
-#ifdef __cpp_concepts
-    static_assert(IsSlabMeta_Arena<SlabMetadata>);
```

Comment:
Is this concept now redundant?

Reply:
Good catch.  Should be; I'll remove it as part of an amended "deorbit" commit.